src/modules_dist.cpp

```diff
@@ -831,7 +831,6 @@ void vinyl_audio_module::post_instantiate(uint32_t sr)
     settings = new_fluid_settings();
     fluid_settings_setnum(settings, "synth.sample-rate", sr);
     fluid_settings_setint(settings, "synth.polyphony", 32);
-    fluid_settings_setint(settings, "synth.midi-channels", _synths);
```

Comment:
I know `_synths` is 7 now, but imagine it gets larger... To make it future proof, what about using `std::max(16, _synths)` instead?

Reply:
That sounds like a good idea. I'll look into it.

Reply:
@zonkmachine Friendly reminder that this comment is still unresolved. Would be really cool to get this PR merged.

Reply:
Rebased on master and pushed the suggested change.

Caveat! I'm on a new build and currently failing to install this correctly so the changes hasn't been tested.